### DISCUSSION TITLE
When can we expect .NET 8 SR 1?

### DISCUSSION THREAD

**Discussion by mjo151** [dotnet/maui#19810] [General]
I'm hoping someone from the Maui team can provide some insight into when we can expect .NET 8 SR 1? It looks like this milestone was completed a while ago. I guess I assumed we would get a service release on a patch Tuesday, but two of those have come and gone since the .NET 8 GA release. I am waiting on some Android fixes that are in this SR, and I believe a number of others are as well.

Side note: Maui has improved greatly in .NET 8, so I appreciate the hard work and look forward to additional updates.

**Comment by HuaFangYun** [comment]
I think the maui update has always been a mystery. I think the SR1 update seems to have been updated to VS17.9. Does it mean to wait for the update of 17.9 to update SR1, expect a regular update, keep learning and using maui, and hope that maui will become better?

**Comment by mjo151** [comment]
@davidortinau Can you provide some insight into when .NET 8 SR1 might be ready? After about a year of development, I am finally ready to release my Maui app but need some fixes that are supposed to be in this service release.
